docs/0.5/references/cli/splinter-circuit-remove-proposal.1.md

```diff
@@ -0,0 +1,84 @@
+% SPLINTER-CIRCUIT-REMOVE-PROPOSAL(1) Cargill, Incorporated | Splinter Commands
```

Comment:
Please add a commit to add a link to this page the CLI Command Reference page

Reply:
Added a commit to add a link to this as well as the circuit-delete commands